subprojects/shipkit/src/main/groovy/org/shipkit/internal/util/GitHubApi.java

```diff
@@ -78,3 +78,5 @@ private String call(String method, HttpsURLConnection conn) throws IOException {
         }
     }
 }
+//https://api.github.com/repos/mstachniuk/mockito/releases
```

Comment:
let's also remove this comment or add it as a more general one to the doc if needed